src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+
+import com.spikes2212.command.DashboardedSubsystem;
+import com.spikes2212.dashboard.Namespace;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import frc.robot.RobotMap;
+
+import java.util.function.Supplier;
+
+public class Leds extends DashboardedSubsystem {
+
+    private final Namespace colorNamespace = namespace.addChild("led colors");
+
+    public static final int NUMBER_OF_LEDS = 60;
```

Comment:
ordering is really bad all over this class. this should be private and on top (static before non-static)